dvc/repo/plots/template.py

```diff
@@ -107,13 +107,15 @@ def _check_field_exists(data, field):
             raise NoFieldInDataError(field)
 
 
-class DefaultLinearTemplate(Template):
+class DefaultTemplate(Template):
```

Comment:
I would consider dropping this one at some point, for sake of interactive `linear` to become the default, though one thing that is stopping me from proposing it right away is the simplicity of the default plot, which definitely makes the learning curve for custom plots smoother.